src/etc/inc/config.inc

```diff
@@ -49,7 +49,11 @@ final class product
 
         /* boot detection */
         $bootflag = '/var/run/booting';
-        @touch($bootflag);
+
+        if (! file_exists($bootflag)) {
```

Comment:
can you try to adhere to PSR12 style? If you have os-debug(or os-debug-devel) installed you can enforce it via:

# make style-fix

from core repository root dir.

Reply:
Sure, sorry about that
It's coding style habits from the phpMyAdmin coding standard

Reply:
Hi @fichtner 
I do not understand how this should work, it throws missing separator `Makefile:29: *** séparateur manquant. Arrêt.`.
It complains at the line where there is an include (`Mk/defaults.mk`), if I comment it the next line starting with a dot is unhappy.

So I tried to run `../../../mariadb-mysql-kbs/vendor/bin/phpcs --standard=ruleset.xml /mnt/Dev/@williamdes/@forked-repos/@opnsense/core/src/etc/inc/config.inc` and it did output some invalid rules errors

After more research I found out that on Debian you need to install `php-codesniffer`.

So I ran `phpcs --standard=ruleset.xml src/etc/inc/config.inc`

And nothing did pop out for the lines you mentioned. `phpcbf` even said `No fixable errors were found`.

I am using a fresh Debian 12.

I bet the operators spacing could be wrong, but phpcs seems to not report it.

Can you help me ?